cromwell-drs-localizer/src/main/scala/drs/localizer/accesstokens/AccessTokenStrategy.scala

```diff
@@ -0,0 +1,5 @@
+package drs.localizer.accesstokens
+
+trait AccessTokenStrategy {
+  def getAccessToken(): String
```

Comment:
What should happen if we can't get a token? Should this just throw? Or should we return a `Try[String]` (or the error-representing monad of your choice)?